### PR TITLE
Updated Exploration faceted search variable names

### PR DESCRIPTION
**Pull request by gkuffel**

### CODE REVIEW

File: niaid.bionimbus.org/portal/gitops.json [hunk 1]
```diff
@@ -245,6 +245,21 @@
...
+        { "field": "status", "name": "Summarized HIV Sero-status"},
+        { "field": "amikacin_res_phenotype", "name": "Amikacin Phenotype"},
+        { "field": "capreomycin_res_phenotype", "name": "Capreomycin Phenotype"},
+        { "field": "isoniazid_res_phenotype", "name": "Isoniazid Phenotype"},
+        { "field": "kanamycin_res_phenotype", "name": "Kanamycin Phenotype"},
+        { "field": "ofloxacin_res_phenotype", "name": "Ofloxacin Phenotype"},
+        { "field": "pyrazinamide_res_phenotype", "name": "Pyrazinamide Phenotype"},
+        { "field": "rifampicin_res_phenotype", "name": "Rifampicin Phenotype"},
+        { "field": "rifampin_res_phenotype", "name": "Rifampin Phenotype"},
+        { "field": "streptomycin_res_phenotype", "name": "Streptomycin Phenotype"},
```
Comment (mfshao) [inline niaid.bionimbus.org/portal/gitops.json]:
```suggestion
        { "field": "streptomycin_res_phenotype", "name": "Streptomycin Phenotype"}
```

File: niaid.bionimbus.org/portal/gitops.json [hunk 2]
```diff
@@ -245,6 +245,21 @@
       "dataType": "subject",
       "nodeCountTitle": "Subjects",
       "fieldMapping": [
+        { "field": "frstdhd", "name": "Year of death" },
+        { "field": "arthxbase", "name": "ART Use Prior to Baseline"},
+        { "field": "bshbvstat", "name": "Baseline HBV Sero-status"},
+        { "field": "bshcvstat", "name": "Baseline HCV Sero-status"},
+        { "field": "cd4nadir", "name": "Year of Death"},
```
Comment (mfshao) [inline niaid.bionimbus.org/portal/gitops.json]: does `cd4nadir` and `frstdhd` all means `Year of death`?

Comment (gkuffel) [reply niaid.bionimbus.org/portal/gitops.json]: Whoops. No. How can I change?

Comment (mfshao) [reply niaid.bionimbus.org/portal/gitops.json]: you can change in your local branch and push to this remote branch again. But remember to `git pull` first on your local branch ;)

File: niaid.bionimbus.org/portal/gitops.json [hunk 3]
```diff
@@ -245,6 +245,21 @@
       "dataType": "subject",
       "nodeCountTitle": "Subjects",
       "fieldMapping": [
+        { "field": "frstdhd", "name": "Year of death" },
```
Comment (mfshao) [inline niaid.bionimbus.org/portal/gitops.json]:
```suggestion
        { "field": "frstdhd", "name": "Year of Death" },
```

**Pull request merged**